app/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,9 @@
         android:xlargeScreens="true" />
 
     <uses-permission android:name="android.permission.REQUEST_DELETE_PACKAGES"/>
+    <uses-permission android:name="android.permission.GET_PACKAGE_SIZE" />
+    <uses-permission android:name="android.permission.GET_PACKAGE_INFO" />
+    <uses-permission android:name="android.permission.GET_SIGNING_CERTIFICATES" />
```

Comment:
Why do we need these permissions ?

Reply:
No, I was testing something so I added them, will remove them.